jedi_language_server/constants.py

```diff
@@ -1,4 +1,19 @@
 """Constants."""
 
+from lsprotocol.types import SemanticTokenTypes
+
 MAX_CONCURRENT_DEBOUNCE_CALLS = 10
 """The maximum number of concurrent calls allowed by the debounce decorator."""
+
+_token_types = list(SemanticTokenTypes)
+
+SEMANTIC_TO_TOKEN_ID = {
+    "module": _token_types.index(SemanticTokenTypes.Namespace),
+    "class": _token_types.index(SemanticTokenTypes.Class),
+    "function": _token_types.index(SemanticTokenTypes.Function),
+    "param": _token_types.index(SemanticTokenTypes.Parameter),
+    "statement": _token_types.index(SemanticTokenTypes.Variable),
+    "property": _token_types.index(SemanticTokenTypes.Property),
+}
+
+SUPPORTED_SEMANTIC_TYPES = list(map(lambda t: t.value, _token_types))
```

Comment:
Are these really the supported types? Or are the supported types actually the ones present in the `SEMANTIC_TO_TOKEN_ID`?

Based on my reading, it looks like our "supported types" are really just the values in `SEMANTIC_TO_TOKEN_ID`... (keys are the jedi strings, right?)

Reply:
The values of `SEMANTIC_TO_TOKEN_ID` are indices into `SUPPORTED_SEMANTIC_TYPES`, so they have to align, but it can be rewritten to only return supported types.